features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,34 @@
+Given /^(?:|I )am on "(.+)"$/ do |page_name|
+  if page_name == "landing page"
+    visit landing_path
+  end
+  if page_name == "home page"
+    visit root_path
+  end
+end
+
+Then /I should see a sign in button/ do
+  expect(find_button("login-button").value).to eq("Placeholder Login")
```

Comment:
Perhaps just look for the existence of a "login-button"? The value will probably not be "Placeholder Login" forever and as it is we'll have to go back and edit this later anyway.

```suggestion
  expect(page).to have_button('login-button')
```